backend-ktor/src/main/kotlin/me/zwsmith/plugins/recipes/RequestData.kt

```diff
@@ -0,0 +1,34 @@
+package me.zwsmith.plugins.recipes
+
+import kotlinx.serialization.Serializable
+import me.zwsmith.plugins.QuantifiedIngredient
+
+@Serializable
+data class RecipeInput(
```

Comment:
These requests should be colocated with the controller/service defining them, as they are part of their public APIs